lib/genre.rb

```diff
@@ -0,0 +1,43 @@
+class Genre
+
+  attr_accessor :name, :musiclibrarycontroller, :musicimporter
```

Comment:
same here. Are all of these needed?

Reply:
Got rid of all of them except for `:name` .